adder/ipfsadd/add.go

```diff
@@ -283,13 +244,80 @@ func (adder *Adder) addNode(node ipld.Node, path string) error {
 	return nil
 }
 
-// AddFile adds the given file while respecting the adder.
-func (adder *Adder) AddFile(path string, file files.Node) error {
-	return adder.addFile(path, file)
+// AddAllAndPin adds the given request's files and pin them.
+// Cluster: we don'pin. Former AddFiles.
+func (adder *Adder) AddAllAndPin(file files.Node) (ipld.Node, error) {
```

Comment:
Why are we naming functions with `pin` if the don't pin?

Reply:
This is a copy of a file from IPFS. The less differences between the two, the easier it is to maintain it. Ideally the diff between to two should be as small as possible. If we change function names, and then IPFS changes function names it becomes harder to figure out what was where.